<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshPrintDialogBase</class>
 <widget class="QDialog" name="MeshPrintDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>メッシュ印刷</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>用紙サイズ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>縮尺</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>出力メッシュ番号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_papers">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>21</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create_mesh">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>100</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>メッシュ作成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pdf_out">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF作成</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="out_mesh_number">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spn_zoom_level">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
